gradle-plugin/it/src/it/probes-groovy-dsl-config/expected/startup/kubernetes.yml

```diff
@@ -0,0 +1,114 @@
+---
+apiVersion: v1
+kind: List
+items:
+- apiVersion: v1
+  kind: Service
+  metadata:
+    annotations:
+      prometheus.io/path: /metrics
+      jkube.io/git-branch: "@ignore@"
+      prometheus.io/port: "9779"
+      prometheus.io/scrape: "true"
+      jkube.io/git-url: "@ignore@"
+      jkube.io/git-commit: "@ignore@"
+    labels:
+      expose: "true"
+      app: probes-groovy-dsl-config
+      provider: jkube
+      version: "@ignore@"
+      group: org.eclipse.jkube.it.gradle.probes.groovy.dsl
+    name: probes-groovy-dsl-config
+  spec:
+    ports:
+    - name: http
+      port: 8080
+      protocol: TCP
+      targetPort: 8080
+    selector:
+      app: probes-groovy-dsl-config
+      provider: jkube
+      group: org.eclipse.jkube.it.gradle.probes.groovy.dsl
+- apiVersion: apps/v1
+  kind: Deployment
+  metadata:
+    annotations:
+      jkube.io/git-url: "@ignore@"
+      jkube.io/git-commit: "@ignore@"
+      jkube.io/git-branch: "@ignore@"
+    labels:
+      app: probes-groovy-dsl-config
+      provider: jkube
+      version: "@ignore@"
+      group: org.eclipse.jkube.it.gradle.probes.groovy.dsl
+    name: probes-groovy-dsl-config
+  spec:
+    replicas: 1
+    revisionHistoryLimit: 2
+    selector:
+      matchLabels:
+        app: probes-groovy-dsl-config
+        provider: jkube
+        group: org.eclipse.jkube.it.gradle.probes.groovy.dsl
+    template:
+      metadata:
+        annotations:
+          jkube.io/git-url: "@ignore@"
+          jkube.io/git-commit: "@ignore@"
+          jkube.io/git-branch: "@ignore@"
+        labels:
+          app: probes-groovy-dsl-config
+          provider: jkube
+          version: "@ignore@"
+          group: org.eclipse.jkube.it.gradle.probes.groovy.dsl
+        name: probes-groovy-dsl-config
+      spec:
+        containers:
+        - env:
+          - name: KUBERNETES_NAMESPACE
+            valueFrom:
+              fieldRef:
+                fieldPath: metadata.namespace
+          - name: HOSTNAME
+            valueFrom:
+              fieldRef:
+                fieldPath: metadata.name
+          image: dsl/probes-groovy-dsl-config:latest
+          imagePullPolicy: IfNotPresent
+          livenessProbe:
```

Comment:
Considering this test-case, I would lean towards removing the actuator dependency.